src/components/LessonsPIA/LessonsPIA.js

```diff
@@ -0,0 +1,103 @@
+                <div className="lessons-pia-description-container">
+                  <div className="lessons-pia-image"><img src={icon} alt="Passion in Action lesson image" width="100%" height="100%"></img></div>
+                  <div className="lessons-pia-description">TODO: Inserire descrizione</div>
+                </div>
+              </div>
+            </Tab>
+            <Tab label="Lezione 2">
+              <div>
+                <h2>Lezione 2</h2>
+                <div className="lessons-pia-description-container">
+                  <div className="lessons-pia-image"><img src={icon} alt="Passion in Action lesson image" width="100%" height="100%"></img></div>
+                  <div className="lessons-pia-description">TODO: Inserire descrizione</div>
+                </div>
+              </div>
+            </Tab>
+            <Tab label="Lezione 3">
+              <div>
+                <h2>Lezione 3</h2>
+                <div className="lessons-pia-description-container">
+                  <div className="lessons-pia-image"><img src={icon} alt="Passion in Action lesson image" width="100%" height="100%"></img></div>
+                  <div className="lessons-pia-description">TODO: Inserire descrizione</div>
+                </div>
+              </div>
+            </Tab>
+            <Tab label="Lezione 4">
+              <div>
+                <h2>Lezione 4</h2>
+                <div className="lessons-pia-description-container">
+                  <div className="lessons-pia-image"><img src={icon} alt="Passion in Action lesson image" width="100%" height="100%"></img></div>
+                  <div className="lessons-pia-description">TODO: Inserire descrizione</div>
+                </div>
+              </div>
+            </Tab>
+            <Tab label="Lezione 5">
+              <div>
+                <h2>Lezione 5</h2>
+                <div className="lessons-pia-description-container">
+                  <div className="lessons-pia-image"><img src={icon} alt="Passion in Action lesson image" width="100%" height="100%"></img></div>
+                  <div className="lessons-pia-description">TODO: Inserire descrizione</div>
+                </div>
+              </div>
+            </Tab>
+          </Tabs>
+        </div>
+      </div> /* End external div */
+    )
+  }
+}
+
+class Tabs extends React.Component{
+  state = {
+    activeTab: this.props.children[0].props.label
+  }
+  changeTab = (tab) => {
+    this.setState({activeTab: tab});
+  };
+  render () {
+    let content;
+    let buttons = [];
+    return (
+      <div>
+        {React.Children.map(this.props.children, child => {
+          buttons.push(child.props.label)
+          if (child.props.label === this.state.activeTab) content = child.props.children
+        })}
+        <TabButtons activeTab={this.state.activeTab} buttons={buttons} changeTab={this.changeTab}/>
+        <div className="lessons-pia-tab-content">{content}</div>
+      </div>
```

Comment:
Questa parte qui non serve, è una complicazione logica, passando i dati tramite props prendi direttamente l'elemento che ti serve renderizzare in base allo stato, senza renderizzarli tutti e senza andare a cercare quello che effettivamente ti serve, all'inizio non è troppo intuitivo, ma con il tempo diventerà più facile lavorare con oggetti JavaScript e renderizzarli in HTML.